.github/workflows/artifacts-build.yml

```diff
@@ -195,36 +194,349 @@ jobs:
         echo "https://storage.googleapis.com/icu4x-pr-artifacts/gha/${{ github.sha }}/wasm-demo/index.html"
         echo "::endgroup::"
 
+  bench-perf:
```

Comment:
Continuing discussion from #3070:

> Putting the post-merge benchmark jobs into the Artifacts Build workflow doesn't seem clean

- @echeran 

Benchmarking is not primarily about building an artifact, but it does build an artifact (the HTML page) that we want to include in the publish job, so it needs to be in the same workflow as the other artifact-build jobs. Maybe we should rename this to 'Artifacts & Bench'? Alternatively, as you proposed, we call this 'Post merge' and either (a) just live with it that FFI doc and rustdoc run on every commit, or (b) not run them on every commit (I'm not sure how much value these add).

Reply:
The current name is fine and makes sense -- dashboards are artifacts, too.